libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveComponentRegistration.cs

```diff
@@ -54,6 +54,12 @@ public override IEnumerable<TypeRegistration> GetTypes()
             yield return new TypeRegistration<OnTypingActivity>(OnTypingActivity.DeclarativeType);
             yield return new TypeRegistration<OnHandoffActivity>(OnHandoffActivity.DeclarativeType);
 
+            yield return new TypeRegistration<OnAsk>("Microsoft.OnAsk");
```

Comment:
>"Microsoft.OnAsk" [](start = 53, length = 17)

please make these public const string on the types, look at the rest of the definitions.

Reply:
Done

---
In reply to: [360160766](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360160766) [](ancestors = 360160766)